### DISCUSSION TITLE
Suppressing  the notification

### DISCUSSION THREAD

**Discussion by kamaraj-muthupandian** [community/community#40901] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Team,

Whenever we are removing and adding user into github teams, we are receiving email notification as below. How to Suppress the notification from the org level.

`You’ve been removed from the "test1" team on the "org-name" organization.

Cheers & Octocats,

GitHub Support
`

**Comment by ossanna16** [comment]
Hi there @kamaraj-muthupandian and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
